src/articles/subjects/science/chemistry/topic-1/1-atoms-what-are-they.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Atoms: what actually are those tiny things?"
+date: "2019-11-09"
+description: "Every substance is made up of small particles called atoms. Atoms are roughly 0.1 nanometers in radius - that's pretty small!"
+subject: "chemistry"
+subjectGroup: "science"
+topic: "topic 1 - atomic structure and the periodic table"
+---
+
+# What is an atom?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)